dotcom-rendering/webpack/webpack.config.server.js

```diff
@@ -43,7 +43,11 @@ module.exports = {
 		...(DEV
 			? [
 					nodeExternals({
-						allowlist: [/^@guardian/],
+						allowlist: [
+							/^@guardian/,
+							// this project is ESM-only and throws an error when bundled
```

Comment:
Doesn't the `allowlist` specifically contain modules that **will** be bundled? As in, `nodeExternals` will remove `node_modules` from the bundle, but the `allowlist` means you put specific ones back in?

Should the comment read:

```suggestion
							// this project is ESM-only and throws an error when not bundled
```

Let me know if I've misunderstood!

Reply:
Yes, you're right!